storage_service/storage_service/settings/base.py

```diff
@@ -181,6 +189,39 @@ def get_env_variable(var_name):
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#authentication-backends
 
 AUTHENTICATION_BACKENDS = ["django.contrib.auth.backends.ModelBackend"]
+
+try:
+    PASSWORD_MINIMUM_LENGTH = int(environ.get("SS_AUTH_PASSWORD_MINIMUM_LENGTH", 8))
+except ValueError:
+    PASSWORD_MINIMUM_LENGTH = 8
+
+AUTH_PASSWORD_VALIDATORS = [
+    {
+        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
+        "OPTIONS": {"min_length": PASSWORD_MINIMUM_LENGTH},
+    }
+]
+
+# Additional password validation is enabled by default. To disable, set the
+# following AUTH_PASSWORD_* settings to false in the environment.
+if not is_false(environ.get("SS_AUTH_PASSWORD_COMMON_VALIDATION", "")):
+    AUTH_PASSWORD_VALIDATORS += [
+        {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"}
+    ]
```

Comment:
`AUTH_PASSWORD_VALIDATORS.append({ ... })` would be a little more idiomatic here unless there's a reason I am missing for not using it? It will also reduce the number of lines a little.

Reply:
`.append()` is certainly more idiomatic and I'm happy to switch it - we seem to use `+=` more commonly in the rest of this module and the Dashboard's `base.py` settings module but I don't see any reason why that needs to be the case or continue. I'm curious how it will reduce the number of lines though? Unless I'm missing something the lines required for the two seem the same, i.e.:

```
AUTH_PASSWORD_VALIDATORS.append(
    {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"}
)
```

vs

```
AUTH_PASSWORD_VALIDATORS += [
    {"NAME": "django.contrib.auth.password_validation.CommonPasswordValidator"}
]
```

Reply:
> I'm curious how it will reduce the number of lines though?

In hindsight - I've no idea what made me think it was fewer lines sorry for the additional noise! :innocent: